google_api.py

```diff
@@ -203,40 +207,42 @@ def gmail_get_messages_from_push(self):
         if 'messages' in response.json():
             self.new_gmail_messages = response.json()['messages']
 
+    def _dissect_message(self,message_payload: str) -> Tuple[str,str,str]:
```

Comment:
I extracted this method for testing, but also to separate logic code from api integration code.